File: inject.js
const fs = require('fs');
const html = fs.readFileSync('temp_sections.html', 'utf8');
const file = 'about.html';
let content = fs.readFileSync(file, 'utf8');
const target = '  <footer class="footer" id="quick-links">';
content = content.replace(target, html + '\n' + target);
fs.writeFileSync(file, content, 'utf8');
console.log('done');

File: inject.py
import os

with open('temp_sections.html', 'r', encoding='utf-8') as f:
    html = f.read()

with open('about.html', 'r', encoding='utf-8') as f:
    content = f.read()

target = '  <footer class="footer" id="quick-links">'
content = content.replace(target, html + '\n' + target)

with open('about.html', 'w', encoding='utf-8') as f:
    f.write(content)

print('done')
